### ISSUE TITLE
himmelblaud stops working after suspend

### ISSUE THREAD

**Issue by Nidhoggrr** [himmelblau-idm/himmelblau#155]
Hi,
after suspend himmelblaud seems to stop working. Getting any logs is impossible, as I can not login anymore :-)
Simply workaround:
add After=[...]suspend.target to himmelblaud.service file

```
[Unit]
Description=Himmelblau Authentication Daemon
After=chronyd.service ntpd.service network-online.target suspend.target
```

**Comment by dmulder** [comment]
Thanks for the report! I'll get this fixed in Tumbleweed.

**Comment by dmulder** [comment]
@Firstyear this might effect Kanidm also, IIUC.

**Comment by dmulder** [comment]
Fixed in 0.4.3, which should land in Tumbleweed soon.

**Comment by Firstyear** [comment]
Thanks, submitted to Kanidm as well.
